.github/workflows/build.yml

```diff
@@ -96,6 +97,8 @@ jobs:
     - name: Test app4triqs
       env:
         DYLD_FALLBACK_LIBRARY_PATH: /usr/local/opt/llvm/lib
+        LD_PRELOAD: /usr/lib/x86_64-linux-gnu/libSegFault.so
```

Comment:
Using `libSegFault` is debatable.

Reply:
Would this have any implications in the absence of segfaults?

Reply:
Essentially the entire constructor boils down to `sigaction (SIGSEGV, &sa, NULL);` where `sa` is the custom segfault handler. So in the absence of `SIGSEGV` this will still consume one additional slot on the signal stack and might clash or otherwise interact with other signal handlers, although I don't think that TRIQS uses POSIX signals for anything.